app/components/blacklight/document/page_header_component.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+module Blacklight
+  module Document
+    # Render the start over and prev/next displays
+    class PageHeaderComponent < Blacklight::Component
+      attr_reader :document, :blacklight_config, :search_context, :search_session
+
+      def initialize(document:, search_context:, search_session:)
+        super
+        @search_context = search_context
+        @search_session = search_session
+        @document = document
+      end
+
+      def render?
+        search_context.present? || search_session.present? || has_header_tools?
+      end
+
+      def before_render
+        @blacklight_config = helpers.blacklight_config
```

Comment:
Do we need this `before_render` method?  Can we just delegate `blacklight_config` to `helpers`?

Reply:
We did not. I didn't know you could delegate in components without causing errors. Thanks for the tip.